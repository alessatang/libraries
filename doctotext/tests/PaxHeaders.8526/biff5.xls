29 mtime=1376895280.82731991
30 atime=1389151481.245853895
30 ctime=1389151481.245853895
