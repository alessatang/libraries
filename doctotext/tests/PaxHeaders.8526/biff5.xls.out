30 mtime=1379553523.499589078
30 atime=1389151481.245853895
30 ctime=1389151481.246853891
